engines/director/lingo/xlibs/jwxini.cpp

```diff
@@ -25,6 +25,7 @@
  * Gahan Wilson's Ultimate Haunted House
  * Wellen, Wracks und Wassermaenner
  *
+ * Karma: Curse of the 12 Caves
```

Comment:
Nom please remove this, since we will have to clean it up once I enable collision handling.

Reply:
Done.